src/components/organisms/Header/Header.tsx

```diff
@@ -19,6 +14,9 @@ const HeaderWrapper = styled.div`
   background-size: cover;
   background: ${({ theme }) => theme.main.colors.purpleDark};
   margin-top: -50px;
+  .icon {
```

Comment:
I'm pretty sure if you set the color of the containing element, the icon should inherit this?